packages/form/src/ux-submit-attribute.ts

```diff
@@ -1,6 +1,7 @@
-import { inject } from 'aurelia-dependency-injection';
+import { inject, customAttribute } from 'aurelia-framework';
```

Comment:
I think we better take dependency on the core module instead of this module. It would be custom attribute from aurelia templating.

Reply:
fixed with https://github.com/aurelia/ux/pull/248/commits/91d7d344f65405a626f9c98cc6346059d0d407c8